src/test/java/de/tum/in/www1/artemis/service/FileServiceTest.java

```diff
@@ -265,38 +260,38 @@ void testPublicPathForActualTempFilePath() {
 
     @Test
     void testPublicPathForActualPath_shouldThrowException() {
-        Exception exception = assertThrows(FilePathParsingException.class, () -> {
+        assertThatExceptionOfType(FilePathParsingException.class).isThrownBy(() -> {
             Path actualFileUploadPath = Path.of(FilePathService.getFileUploadExercisesFilePath());
             fileService.publicPathForActualPath(actualFileUploadPath.toString(), 1L);
-        });
-        assertThat(exception.getMessage()).startsWith("Unexpected String in upload file path. Exercise ID should be present here:");
 
-        exception = assertThrows(FilePathParsingException.class, () -> fileService.publicPathForActualPath(Path.of("asdasdfiles", "file-asd-exercises").toString(), 1L));
-        assertThat(exception.getMessage()).startsWith("Unknown Filepath:");
+        }).withMessageStartingWith("Unexpected String in upload file path. Exercise ID should be present here:");
+
+        assertThatExceptionOfType(FilePathParsingException.class).isThrownBy(() -> fileService.publicPathForActualPath(Path.of("asdasdfiles", "file-asd-exercises").toString(), 1L))
+                .withMessageStartingWith("Unknown Filepath:");
     }
 
     @Test
     void testReplaceVariablesInFileRecursive_shouldThrowException() {
-        Exception exception = assertThrows(RuntimeException.class, () -> fileService.replaceVariablesInFileRecursive("some-path", new HashMap<>()));
-        assertThat(exception.getMessage()).endsWith("should be replaced but the directory does not exist.");
+        assertThatRuntimeException().isThrownBy(() -> fileService.replaceVariablesInFileRecursive("some-path", new HashMap<>()))
+                .withMessageEndingWith("should be replaced but the directory does not exist.");
     }
 
     @Test
     void testNormalizeLineEndingsDirectory_shouldThrowException() {
-        Exception exception = assertThrows(RuntimeException.class, () -> fileService.normalizeLineEndingsDirectory("some-path"));
-        assertThat(exception.getMessage()).endsWith("should be normalized but the directory does not exist.");
+        assertThatRuntimeException().isThrownBy(() -> fileService.normalizeLineEndingsDirectory("some-path"))
+                .withMessageEndingWith("should be normalized but the directory does not exist.");
     }
 
     @Test
     void testConvertToUTF8Directory_shouldThrowException() {
-        Exception exception = assertThrows(RuntimeException.class, () -> fileService.convertToUTF8Directory("some-path"));
-        assertThat(exception.getMessage()).endsWith("should be converted to UTF-8 but the directory does not exist.");
+        assertThatRuntimeException().isThrownBy(() -> fileService.convertToUTF8Directory("some-path"))
+                .withMessageEndingWith("should be converted to UTF-8 but the directory does not exist.");
     }
 
     // TODO: either rework those tests or delete them
```

Comment:
Worth checking on this TODO?

Reply:
I would leave it for now and work on it in a future PR